src/debian/iot/arm32v7/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM arm32v7/debian:latest
+
+# Dependencies for downloading and running the installation script of libgpiod
+RUN apt-get update \
+    && apt-get install -y \
+            wget \
```

Comment:
Docker best practices suggest alphabetizing apt-get package lists.  It helps the readability.  This is definitely the case for some of our amd64 Dockerfiles where the number of packages is very large.